app/visit/exhibit-card.test.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { describe, it, expect } from 'vitest';
+
+import ExhibitCard from './exhibit-card';
+
+import { exhibits } from './page';
+
+describe('Header Component', () => {
```

Comment:
```suggestion
describe('Exhibit Card', () => {
```

We could add a few more tests, to make sure the elements are rendered as expected. Especially with with the absolutely positioned overlays, it would be nice to ensure the cards are clickable and the text can be read.

Reply:
@AliceR I added more tests.